assets/stylesheets/components/_media-objects.scss

```diff
@@ -0,0 +1,172 @@
+/**
+** Media Objects
+** 1. Base layout
+** 2. Text
+** 3. Images
+** 4. Overrides
+** 5. Connect-specific layout
+**/
+
+
+
+
+// Base layout
+.media {
+  border-top: 1px solid rgba(darken($cr-gray-lighter, 10), .4);
+  cursor: pointer;
+  display: flex;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+
+  > div:last-child {
+    margin-left: 1rem;
+  }
+
+  .media-body {
+    flex-grow: 1;
+    width: initial;
+  }
+}
+
+
+
+
+// Text
+.media-heading {
+  @extend .component-header;
+  color: $cr-gray-dark;
+  margin-bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &.media-heading-link {
+    color: $cr-cyan;
+  }
+}
+
+.say-hi {
+  .media-heading &,
+  .media-heading-stacked & {
+    color: $cr-gray-dark;
+    font-family: $base-font-face;
+    font-size: $font-size-base;
+    text-transform: initial;
+    padding-bottom: .5rem;
+  }
+}
+
+.media-heading-stacked {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  h1 {
+    @extend .component-header;
+    color: $cr-white;
+    font-size: $font-size-h1;
+    margin: 0;
+  }
+}
+
+.media-meta {
+  @extend .control-label;
+  color: $cr-gray-light;
+
+  .joined_by {
+    margin-left: .5rem;
+  }
+}
+
+
+
+
+// Images
+// Create custom .img-size-* classes at 16px base
+@for $i from 1 through 6 {
+  .img-size-#{$i} {
+    $dimensions: 16px;
+
+    height: $dimensions * $i;
+    width: $dimensions * $i;
+  }
+}
+
+.media-img-wrapper {
+  margin-top: .25rem;
+  position: relative;
+
+  .media & {
+    display: flex;
+    flex-direction: row-reverse;
+    width: 96px;
+  }
+
+  .img-circle {
+    border-color: rgba($cr-gray-lighter, .4);
+    border-style: solid;
+    border-width: 2px;
+  }
+}
+
+.media-object-default {
+  background-image: url('//crossroads-media.imgix.net/images/avatar.svg');
+  background-position: center;
+  background-size: cover;
+}
+
+.media-object-overlay {
+  background-position: center;
+  background-size: cover;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+
+
+
+// Overrides
+.media {
+  &.site {
+    cursor: initial;
+  }
+}
+
+
+
+
+// 🚨 Layout for list view - connect-specific?
+.connect-white-bg {
```

Comment:
@zarahzachz: I think that's totally fine. Is there a way we can use a more generic name and treat it like a variation on a media object rather than specific to one app?

I agree it's more likely people keep adding CSS to the project if you leave it there, but it seems weird to mention connect in crds-styles.

This is slightly off-topic, but I do think any styles that remain in connect should be split out into partials so we aren't just dumping everything in application.scss like we have for the last sprint.